src/shared/components/input/Input.scss

```diff
@@ -1,23 +1,60 @@
 @import 'src/styles/common';
 
 .input {
-  display: inline-block;
-  width: 100%;
+  &::placeholder {
+    color: $dark-grey;
+    font-weight: $light;
+  }
 
   &:focus,
   &:active {
     outline: none;
+    color: $black;
+    font-weight: $normal;
```

Comment:
Is font-weight necessary? Isn't it the default?

Reply:
Removed these ✅